app/src/main/res/values-ceb/strings.xml

```diff
@@ -0,0 +1,543 @@
+    <string name="browser_menu_bookmark">i-Bookmark</string>
+    <!-- Content description (not visible, for screen readers etc.): Un-bookmark the current page -->
+    <string name="browser_menu_edit_bookmark">Usba ang bookmark</string>
+
+
+    <!-- Browser menu button that opens the addon manager -->
+    <string name="browser_menu_add_ons">Mga Add-on</string>
+    <!-- Browser menu button that opens the addon extensions manager -->
+    <string name="browser_menu_extensions">Mga Extension</string>
+    <!-- Text displayed when there are no add-ons to be shown -->
+    <string name="no_add_ons">Wala’y mga add-on dinhi</string>
+    <!-- Browser menu button that sends a user to help articles -->
+    <string name="browser_menu_help">Tabang</string>
+    <!-- Browser menu button that sends a to a the what's new article -->
+    <string name="browser_menu_whats_new">Unsay Bag-o</string>
+    <!-- Browser menu button that opens the settings menu -->
+    <string name="browser_menu_settings">Mga Setting</string>
+    <!-- Browser menu button that opens a user's library -->
+    <string name="browser_menu_library">Library</string>
+    <!-- Browser menu toggle that requests a desktop site -->
+    <string name="browser_menu_desktop_site">Desktop site</string>
+    <!-- Browser menu toggle that adds a shortcut to the site on the device home screen. -->
+    <string name="browser_menu_add_to_homescreen">Dugang sa Home screen</string>
+    <!-- Browser menu toggle that installs a Progressive Web App shortcut to the site on the device home screen. -->
+    <string name="browser_menu_install_on_homescreen">i-Install</string>
+    <!-- Menu option on the toolbar that takes you to synced tabs page-->
+    <string name="synced_tabs">Na-sync nga mga tab</string>
+    <!-- Content description (not visible, for screen readers etc.) for the Resync tabs button -->
+    <string name="resync_button_content_description">i-Sync usab</string>
+    <!-- Browser menu button that opens the find in page menu -->
+    <string name="browser_menu_find_in_page">Pangitaa sa page</string>
+    <!-- Browser menu button that creates a private tab -->
+    <string name="browser_menu_private_tab">Private tab</string>
+    <!-- Browser menu button that saves the current tab to a collection -->
+    <string name="browser_menu_save_to_collection_2">i-Save sa collection</string>
+    <!-- Browser menu button that open a share menu to share the current site -->
+    <string name="browser_menu_share">i-Share</string>
+    <!-- Share menu title, displayed when a user is sharing their current site -->
+    <string name="menu_share_with">i-Share sa</string>
+    <!-- Browser menu button shown in custom tabs that opens the current tab in Fenix
+        The first parameter is the name of the app defined in app_name (for example: Fenix) -->
+    <string name="browser_menu_open_in_fenix">Ablihi sa % 1 $ s</string>
```

Comment:
```suggestion
    <string name="browser_menu_open_in_fenix">Ablihi sa %1$s</string>
```

Reply:
I left [a fix](https://pontoon.mozilla.org/ceb/firefox-for-android/mozilla-mobile/fenix/app/src/main/res/values/strings.xml/?search=browser_menu_open_in_fenix&string=197456) for this in Pontoon, but we need someone from l10n to approve it while Delphine is on PTO.